src/test/config.sol

```diff
@@ -305,7 +307,7 @@ contract Config {
             line:         0,
             dust:         7_500,
             pct:          9_25,
-            mat:          14500,
+            mat:          15000,
```

Comment:
I'm not sure why this needed to be changed.

Reply:
The changes are coming from the `lerp` executed in the 2024-10-04 spell and only finished on the weekend. Current config tests are not designed to take end lerp values into account: https://github.com/makerdao/spells-mainnet/blob/219715be3e4653ff53cd4d6a47b8313fa6720f4e/archive/2024-10-04-DssSpell/DssSpell.sol#L270-L280